features/bevragen/persoon/verblijfplaats/dev/Informatievragen-fields-gba.feature

```diff
@@ -44,46 +45,58 @@ Functionaliteit: vertalen van gevraagde samengestelde of afgeleide verblijfplaat
       |                      | Boterondiep | 31         | 3077AW   | Rotterdam  | 0599010000208579                 | 0599200000219678              |                              |
+      | .straat              | .           | .          | .        | .          | .                                | .                             |                              |
+      | .locatiebeschrijving | .           | .          | .        | .          | .                                | .                             |                              |
+      |                      | .           | .          | .        | .          | .                                | .                             |                              |
       | .straat              |             |            |          |            |                                  |                               | Woonboot in de Grotere Sloot |
       | .locatiebeschrijving |             |            |          |            |                                  |                               | Woonboot in de Grotere Sloot |
       |                      |             |            |          |            |                                  |                               | Woonboot in de Grotere Sloot |
+      | .straat              |             |            |          |            |                                  |                               | .                            |
+      | .locatiebeschrijving |             |            |          |            |                                  |                               | .                            |
+      |                      |             |            |          |            |                                  |                               | .                            |
 
-  Rule: wanneer een veld van verblijfplaats m.b.v. de alias 'verblijfplaatsBinnenland' wordt gevraagd en het veld 'land' heeft de waarde '0000' dan wordt dat veld geleverd zodat het type 'VerblijfplaatsOnbekend' kan worden bepaald.
+  Rule: als de alias verblijfplaatsBinnenland of één of meerdere velden van verblijfplaatsBinnenland wordt gevraagd en er zijn geen waarden in de binnenlandsadres velden (dus ook geen standaardwaarden) dan wordt er geen verblijfplaats object teruggeleverd.
 
-    Scenario: met fields wordt gevraagd naar een adresveld bij een onbekend adres m.b.v. de alias verblijfplaatsBinnenland
-      Gegeven de persoon met burgerservicenummer '000000292' heeft de volgende 'verblijfplaats' gegevens
-      | naam                                   | waarde   |
-      | land (13.10)                           | 0000     |
+# Het is de vraag of dit scenario nog iets toevoegt aan het volgende.
+    Scenario: met fields wordt gevraagd naar een adresveld bij adresvelden zonder waarde m.b.v. de alias verblijfplaatsBinnenland
```

Comment:
De titel klopt in ieder geval niet met de body van de scenario. In dit scenario heeft de persoon helemaal geen verblijfplaats ipv geen adresvelden/adresvelden zonder waarde.